VAMobile/src/testUtils.tsx

```diff
@@ -189,6 +190,24 @@ function render(ui, { preloadedState, navigationProvided = false, queriesData, .
         },
       },
     })
+    queryClient.setQueryData(authorizedServicesKeys.authorizedServices, {
+      appeals: true,
+      appointments: true,
+      claims: true,
+      decisionLetters: true,
+      directDepositBenefits: true,
+      directDepositBenefitsUpdate: true,
+      disabilityRating: true,
+      genderIdentity: true,
+      lettersAndDocuments: true,
+      militaryServiceHistory: true,
+      paymentHistory: true,
+      preferredName: true,
+      prescriptions: true,
+      scheduleAppointments: true,
+      secureMessaging: true,
+      userProfileUpdate: true,
+    })
```

Comment:
Since we're referencing authorized services in multiple queries, this was needed to prevent multiple unit tests from failing because the values for these weren't mocked. These values can still be altered if need be by using setting the `queriesData` param just like we do with other data